tests/unit_tests/core/util/byte_stream.cc

```diff
@@ -181,44 +181,44 @@ BOOST_AUTO_TEST_CASE(Bits64Test)
   std::uint64_t value = std::numeric_limits<std::uint64_t>::max();
 
   core::OutputByteStream output(buffer.data(), buffer.size());
-  BOOST_CHECK_NO_THROW(output.Write<std::uint64_t>(value));
-  BOOST_CHECK_THROW(output.Write<std::uint8_t>(0), std::length_error);
+  BOOST_CHECK_NO_THROW(output.write<std::uint64_t>(value));
+  BOOST_CHECK_THROW(output.write<std::uint8_t>(0), std::length_error);
 
   core::InputByteStream input(buffer.data(), buffer.size());
-  BOOST_CHECK_EQUAL(input.Read<std::uint64_t>(), value);
-  BOOST_CHECK_THROW(input.Read<std::uint8_t>(), std::length_error);
+  BOOST_CHECK_EQUAL(input.read<std::uint64_t>(), value);
+  BOOST_CHECK_THROW(input.read<std::uint8_t>(), std::length_error);
 }
 
-BOOST_AUTO_TEST_CASE(AddressToByteVectorIPv4)
+BOOST_AUTO_TEST_CASE(address_to_bytesIPv4)
 {
   boost::asio::ip::address address;
   BOOST_CHECK_NO_THROW(
       address = boost::asio::ip::address::from_string(m_IPv4String));
-  auto const ip = core::AddressToByteVector(address);
+  auto const ip = core::address_to_bytes(address);
   BOOST_CHECK_EQUAL(ip.size(), address.to_v4().to_bytes().size());
   BOOST_CHECK_EQUAL_COLLECTIONS(
-      ip.data(),
-      ip.data() + ip.size(),
-      m_IPv4Array.data(),
-      m_IPv4Array.data() + m_IPv4Array.size());
+      ip.begin(),
+      ip.end(),
+      m_IPv4Array.begin(),
+      m_IPv4Array.end());
   // Reconstruct a new address and check with original
   boost::asio::ip::address_v4::bytes_type bytes;
   std::memcpy(bytes.data(), ip.data(), address.to_v4().to_bytes().size());
   BOOST_CHECK_EQUAL(boost::asio::ip::address_v4(bytes), address.to_v4());
 }
 
-BOOST_AUTO_TEST_CASE(AddressToByteVectorIPv6)
+BOOST_AUTO_TEST_CASE(address_to_bytesIPv6)
```

Comment:
Watchout for your global sed.

Reply:
Right, I missed this. Will update with the other fixups.